### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by davidbrochart**

### CODE REVIEW

File: docs/src/pages/blog/introducing-anywidget.mdx [hunk 1]
```diff
@@ -392,7 +392,7 @@ modern front-end development tools. The [Vite plugin](/en/bundling#development)
 for **anywidget** extends [Vite's](https://vitejs.dev/) development server with precise
 [Hot Module Replacement](https://vitejs.dev/guide/features.html#hot-module-replacement) support for Jupyter Widgets.
 Changes made to the JavaScript source during development are **instantly** reflected
-in the front end without refreshing or re-executing notebook cells.
+in the front-end without refreshing or re-executing notebook cells.
```
Comment (manzt) [inline docs/src/pages/blog/introducing-anywidget.mdx:395]: People seem to have differing opinions about frontend vs front-end vs front end. In this article I use "front-end" when it is used as a compound adjective and "front end" when used as a noun (a recommendation I've seen [several places](https://css-tricks.com/poll-results-front-end-front-end/#aa-the-right-answer-seems-to-be).

File: docs/src/pages/blog/introducing-anywidget.mdx [hunk 2]
```diff
@@ -458,15 +458,15 @@ with the `sync=True` metadata – just like regular Jupyter Widgets!
...
+> without additional installation or setup.
 
 ### Examples
 
 #### Two-Way Data Binding
 
 The `ExampleWidget` demonstrates simple one-way data binding from the Python kernel
 to front-end view. This example is slightly more sophisicated and synchronizes the
-model state between the widget fronend and the Python kernel (i.e., two-way data binding).
+model state between the widget front-end and the Python kernel (i.e., two-way data binding).
```
Comment (manzt) [inline docs/src/pages/blog/introducing-anywidget.mdx:469]: See above, should be "front end".

File: docs/src/pages/blog/introducing-anywidget.mdx [hunk 3]
```diff
@@ -475,7 +475,7 @@ updates the text output any time `count` changes.
 By treating the model as the source of truth, regardless of whether `count` is updated on the
 Python side or the JavaScript side, the count displayed remains correct. You can learn more
 about model view controller (MVC), the IPython widget framework
-the front end relies, in the Jupyter Widgets [documentation](https://ipywidgets.readthedocs.io/en/8.0.2/examples/Widget%20Custom.html#Models-and-views).
+the front-end relies on, in the Jupyter Widgets [documentation](https://ipywidgets.readthedocs.io/en/8.0.2/examples/Widget%20Custom.html#Models-and-views).
```
Comment (manzt) [inline docs/src/pages/blog/introducing-anywidget.mdx:478]: likewise to keep "front end"

**Review by manzt — CHANGES_REQUESTED** [review]
Thank you! Just some minor comments.

File: docs/src/pages/blog/introducing-anywidget.mdx (hunk 1 above)
Comment (davidbrochart) [reply docs/src/pages/blog/introducing-anywidget.mdx]: Oh I didn't know :+1:

**Review by manzt — APPROVED** [review]
Awesome, thanks!

**Pull request merged**